README.md

```diff
@@ -1,70 +1,122 @@
 # 96Boards.org
 
-The 96Boards website is built using the Jekyll static site generator. It is based off the jumbo-jekyll-theme used for all of the Linaro static sites. With the move from Wordpress to Static we have introduced GitHub/Community driven content that allows 96Boards.org users to submit issues about any of the pages on 96Boards.org through the associated Git respository. This GitHub repository is combined with the 96Boards Documentation repo to build the site. Documentation is copied to the `_documentation` collection upon site builds with [bamboo.linaro.org](https://bamboo.linaro.org). 
+![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0)
 
-![Build Status](https://bamboo.linaro.org/plugins/servlet/wittified/build-status/BOARDS-BUILDDEV0) 
+The 96Boards website is built using the [Jekyll](https://jekyllrb.com/) static site generator. It uses the Linaro [jumbo-jekyll-theme](https://github.com/linaro-marketing/jumbo-jekyll-theme) (which is used for all of the Linaro static websites). With the source for the source for the 96Boards.org website available on GitHub in this repo, it makes it easy for community/technical contributors to submit content. The 96Boards documentation repo is synced into the _documentation directory during site builds that occur on our Atlassian Bamboo CI instance.
```

Comment:
"The 96Boards documentation repo is synced into the _documentation directory during site builds that occur on our Atlassian Bamboo CI instance." ... forget to turn off italics?